book/website/community-handbook/communication-channels/slack-start-guide.md

```diff
@@ -83,11 +83,14 @@ If a channel has become too noisy, you can additionally:
 
 * While IM-based conversations can feel very immediate, please don’t feel that you have to respond to messages immediately, nor expect others to respond immediately. We’re all busy, really busy!
 
-
 #### Message Editing & Deletion
 
 * You are allowed to edit your messages at any time. That means if you edit a message after someone replied to it, make it clear that you edited something if it changes the meaning of your message. See the example in the GIF below.
 
+#### Account Management
+
+* To change the email your slack account is attached to, message a member of _The Turing Way_ team, who will have to delete your account first, so you can join with an alternative email.
```

Comment:
To change the email your slack account is attach to, just follow these instructions form Slack help: https://slack.com/intl/en-gb/help/articles/207262907-Change-your-email-address. 
If you need to merge two existing accounts, message message a member of _The Turing Way_ team, who will have to delete your account first, so you can join with your preferred email.